.editorconfig

```diff
@@ -0,0 +1,15 @@
+root = true
+
+[*]
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+indent_style = space
+indent_size = 4
+
+[{Makefile, **.mk}]
+indent_style = tab
+
+[*.go]
+indent_style = tab
```

Comment:
Really? instead of spaces?

Reply:
yes. this is decreed by the golang gods. and no, you can't argue with it...